catalogue/webapp/components/MultipleManifestList/MultipleManifestList.tsx

```diff
@@ -47,45 +46,34 @@ const HiddenContent = styled.div.attrs(props => ({
   }
 `;
 
-type Props = {|
-  buttonText: string,
-  manifests: IIIFManifest[],
-  workId: string,
-  lang: string,
-|};
+type Props = {
+  buttonText: string;
+  manifests: IIIFManifest[];
+  workId: string;
+  lang: string;
+};
 
-const MultipleManifestList = ({
+const MultipleManifestList: FunctionComponent<Props> = ({
   buttonText,
   manifests,
   workId,
   lang,
 }: Props) => {
   const [showHidden, setShowHidden] = useState(false);
-  const wrapperRef = useRef(null);
-  const downloadText = useRef(null);
+  const wrapperRef = useRef<HTMLDivElement>(null);
+
+  const handleClickOutside = event => {
+    if (wrapperRef.current && !wrapperRef.current?.contains(event.target)) {
+      setShowHidden(false);
+    }
+  };
   useEffect(() => {
     window.document.addEventListener('click', handleClickOutside, false);
     return () => {
       window.document.removeEventListener('click', handleClickOutside, false);
     };
   }, []);
 
-  const handleClickOutside = event => {
```

Comment:
Does anyone know why this block exists? afaict the `downloadText` ref doesn't and has never been attached to anything

Reply:
Ping @gestchild - but as far as I can see - I'd agree.